### DISCUSSION TITLE
USACE Requirements for OpenDCS in CWBI:  Public Access to OpenDCS Data

### DISCUSSION THREAD

**Discussion by jbatterman** [opendcs/opendcs#336] [General]
The Missouri River Basin Water Management Office has brainstormed some requirements for what is needed for USACE to operate OpenDCS in the Cloud.  We are looking for feedback on these requirements, and additions from other offices who have different data acquisition processes.  There will be a new Discussion for each requirement.

As OpenDCS is incorporated into CDA, ensure the public does not have read/write access to OpenDCS data (i.e., configurations, platform information, algorithms, etc.).

@MikeNeilson @hoffmanjs @opendcs/opendcs-core-devs @opendcs/opendcs-governance @opendcs/opendcs-triage

**Comment by MikeNeilson** [comment]
There is no intention to incorporate OpenDCS database operations into CDA.

We'll be using https://github.com/opendcs/rest_api as it can operate appropriately.

Additionally, while there is information that should be shared. We *should* make it easy to acquire and use the GOES platform and decodes scripts. That information is public by agreement with NESDIS to use the system.

**Reply by jbatterman** [reply]
Thanks for the clarification @MikeNeilson.  Whatever API it's using, we do want to verify that if the public can access our OpenDCS data, that it is read-only access, and that they cannot access it and modify decoding scripts, etc.

**Reply by MikeNeilson** [reply]
Yep, that is already part of the API that was developed.
